compositions/upbound-aws-provider/lambda/container.yaml

```diff
@@ -45,8 +45,6 @@ spec:
         spec:
           deletionPolicy: Delete
           forProvider:
-            memorySize: 128
-            timeout: 30
```

Comment:
Can we either keep the default timeout here or set the default timeout to 30 in the definition?

Reply:
Ah yes, that was the intention to set a default in the definition here 
https://github.com/csantanapr/crossplane-on-eks/blob/dynatrace-log-forwarder/compositions/upbound-aws-provider/lambda/definition.yaml#L27-L28
It already has the memory 128, timeout 30 should be added